package cz.hyperion.exception;

public class FinishException extends RuntimeException{
}
